diff --git a/src/util/xmlpool/meson.build b/src/util/xmlpool/meson.build
index d0ecec4..1dbfacd 100644
--- a/src/util/xmlpool/meson.build
+++ b/src/util/xmlpool/meson.build
@@ -36,7 +36,7 @@ xmlpool_options_h = custom_target(
   depend_files : _langs_po_files,
 )
 
-if host_machine.system() != 'windows'
-  i18n = import('i18n')
-  i18n.gettext('xmlpool', install : false)
-endif
+# if host_machine.system() != 'windows'
+#   i18n = import('i18n')
+#   i18n.gettext('xmlpool', install : false)
+# endif
